README.md

```diff
@@ -240,6 +241,17 @@ const mapDispatchToProps = (dispatch, props) => {
 };
 ```
 
+The result of the promise returned by `mutateAsync` will be the following object:
+
+| Name | Type | Description |
+|:----|:-----|:-----|
+| status | number | HTTP status code
+| body | object | Response body auto-parsed by SuperAgent
```

Comment:
change to "Parsed response body." prefer not to mention superagent soon (see #23) will support other libs that will provide the same interface